### PR TITLE
Fix command completion

### PR DESCRIPTION
**Pull request by woksin**
Fixes a bug regarding the completion of event handling. Also fixes some of the logging.

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1140654762299587/1179552285238164)

### CODE REVIEW

**Comment by woksin** [comment]
Fixing specs :)

File: Source/Commands.Coordination/Runtime/CommandCoordinator.cs [hunk 1]
```diff
@@ -102,32 +102,32 @@ CommandResult Handle(ICommandContext commandContext, CommandRequest command)
...
-                        _logger.Debug("Command was not successful, rolling back");
+                        _logger.Debug("Command {CommandType} with {Correlation} was not successful, rolling back", command.Type, command.CorrelationId);
                         commandContext.Rollback();
                     }
                 }
             }
             catch (TargetInvocationException ex)
             {
-                _logger.Error(ex, "Error handling command");
+                _logger.Warning(ex, "Error handling command {CommandType} with correlation {Correlation}", command.Type, command.CorrelationId);
```
Comment (smithmx) [inline Source/Commands.Coordination/Runtime/CommandCoordinator.cs:125]: Is this a policy you have in Core?  Why log exceptions as warnings and not as errors?

**Review by joelhoisko — APPROVED** [review]
Looks good to me 👍

File: Source/Events.Handling/EventProcessingCompletion.cs [hunk 2]
```diff
@@ -35,60 +35,73 @@ public EventProcessingCompletion(ILogger logger)
...
             {
-                action();
-
-                _logger.Trace("Waiting for EventHandlers");
-                await waiter.Complete().ConfigureAwait(false);
-                _logger.Trace("EventHandler waiting is done");
+                try
+                {
+                    action();
+                    _logger.Trace("EventProcessingCompletion.PERFORM LOOP Time for eventHandlers {CorrelationId}", correlationId);
```
Comment (smithmx) [inline Source/Events.Handling/EventProcessingCompletion.cs:87]: Can probably lose this log message, it doesn't add anything that the one below doesn't already have.

**Review by smithmx — APPROVED** [review]

**Pull request merged**